db/migrate/20201215124546_add_publisher_whitelist_updates.rb

```diff
@@ -0,0 +1,10 @@
+class AddPublisherWhitelistUpdates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :publisher_whitelist_updates, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :publisher, type: :uuid, index: true, unique: true, null: false
+      t.references :publisher_note, type: :uuid
+      t.boolean :enabled, null: false
+      t.timestamps
```

Comment:
Would you add indexes to the timestamps?

Reply:
Added